english/triathlon/triathlon.md

```diff
@@ -16,3 +16,5 @@ reading reviews.
 Okay, now you realize that this is going to cost you some real money.  You
 have to get some serious gear to compete.  You can't just use an old bike 
 and any pair of sneakers.  You need to put together a budget.
+
+When you sit down to crunch the numbers for your budget, your wife asks, "When's last time you crunched your abs? You know the triathlon is right around the corner.  Suddenly it hits you, you've been avoiding what you're really afrad of. It's a lot easier to buy gear and make plans to train rather than actually training.  Off to the gym you go.
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.